src/Arbitrary.hs

```diff
@@ -638,8 +639,9 @@ instance Arbitrary InitialPassword where
     shrink = gshrink
 
 instance Arbitrary EncryptedPassword where
-    arbitrary = garbitrary
-    shrink    = gshrink
+    arbitrary = mk <$> arb <*> arb
+      where
+        mk salt pass = ScryptEncryptedPassword . getEncryptedPass $ encryptPass' (Salt salt) (Pass pass)
```

Comment:
my mistake.  this is the type for encrypted passwords only, and it is contained in the `UserPass`, where there are other constructors.  i got the two confused.